packages/contracts/test/contracts/unichain/DataUnionTemplate.test.ts

```diff
@@ -431,7 +432,7 @@ describe("DataUnionTemplate", () => {
     })
 
     it("rejects fees that sum above 1.0", async () => {
-        await expect(dataUnionSidechain.setFees(parseEther("0.5"), parseEther("0.6"))).to.be.revertedWith("error_fees")
+        await expect(dataUnionSidechain.setAdminFee(parseEther("0.95"))).to.be.revertedWith("error_adminFee")
     })
 
```

Comment:
Should add test case for "it changes the admin fee to 1-protocolFee on refreshRevenue if the total fees are above 1"